test/tools/libtesteth/FillJsonFunctions.cpp

```diff
@@ -151,7 +151,7 @@ json_spirit::mObject fillJsonWithStateChange(
             log << (*it).second.str();
     }
 
-    clog(VerbosityInfo, "state") << log.str();
+    clog(VerbosityTrace, "state") << log.str();
```

Comment:
I saw some things logged with `LOG(m_logger) << "something something"` - they will comply to logging level we use or these logs are not visible?

Also are logs which we see most often when transaction is executed still on info level? For example:
```
Paying 21 Kwei from sender for gas (21000 gas at 1 wei)
```

Reply:
yes

Reply:
They are in trace level now